README.md

```diff
@@ -1,21 +1,88 @@
-# App Idea Finder
+# Develapp
+
+![](https://img.shields.io/badge/Name-Develapp-brightgreen?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Version-v0.0.1-blueviolet?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Stack-MERN-important?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Team-chingu.v38.geckos.04-informational?style=plastic&labelColor=inactive)
+
+<br>
 
 ## Overview
 
-This app was created for the Chingu Voyage 38. 
+One difficulty in learning how to code is figuring out which project to build to improve your skills. Most tutorials simply build a to-do app, probably because the tutors too having a hard time figuring out which projects to build. Perhaps they just want to keep it simple. Develapp is providing solution to developers and tutors who are looking for project ideas to build and include in their portfolio or tutorials. Develapp is made for you so that you can **always be building**.
+
+This project was built by a remote team during the `chingu` voyage 38.
+
+**Live link:** --- You can find the app [here]()
+
+<br>
+
+## SCREENSHOTS 📸📸📸
+Include screenshots here.
+
+<br>
+
+## FEATURES ✨✨✨
+The features of the `Develapp` app are -
+- users can find project ideas
+- project ideas are grouped into 
+    * beginner
+    * intermediate
+    * advanced
+- users can be redirected to `chingu` project showcase page
+- users can find project ideas by name
+- users can find project ideas by clicking either one of the three categories on the home page
+- and many more
```

Comment:
Do we have more features at this point? I suggest removing this and list all of the current features.

Reply:
No, we don't. Just an indication of more to come. But I can remove it, no problem.